lib/src/model/input_history_item.dart

```diff
@@ -4,7 +4,6 @@ class InputHistoryItem {
   late int pinnedTime;
   bool isLock = false;
 
-  String get textToSingleLine => text.replaceAll("\n", "").replaceAll(" ", "");
   String get createdTimeLabel {
```

Comment:
`textToSingleLine`  is used and cannot be deleted. Deletion is canceled once 🙏 

https://github.com/trashfeed/input_history_text_field/blob/c7fe6d3753023896e53f88e9fdbb3e2ad5740fa5/example/lib/main.dart#L128